pwned_passwords_django/tests/test_api.py

```diff
@@ -77,8 +81,69 @@ def test_empty_response(self):
         with mock.patch('requests.get', request_mock):
             result = api.pwned_password(self.sample_password)
             request_mock.assert_called_with(
-                api.API_ENDPOINT.format(
+                url=api.API_ENDPOINT.format(
                     self.sample_password_prefix
-                )
+                ),
+                timeout=0.6,
+            )
+            self.assertEqual(None, result)
+
+    def test_bad_text(self):
+        """
+        Handle non-numeric count gracefully
```

Comment:
I'd prefer the docstring to be a full sentence with period, and to be worded more like:

"Non-numeric count in API response is handled gracefully."